ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,8 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+int convertMarkTo5PointSystem(int markIn100Point){
+    if(markIn100Point>90) markIn100Point = 90;
```

Comment:
hacky way, try to solve it with such mapping:
{{0, 1}, {30, 2}, {50, 3}, {70, 4}, {90, 5}};